app/commands/helpers/incident_helper.py

```diff
@@ -268,6 +273,31 @@ def save_incident_roles(client, ack, view):
     )
 
 
+def close_incident(client, body, ack):
+    ack()
+    # get the current chanel id and name
+    channel_id = body["channel_id"]
+    channel_name = body["channel_name"]
+
+    # get and update the incident document
+    document_id = ""
+    response = client.bookmarks_list(channel_id=channel_id)
+    if response["ok"]:
+        for item in range(len(response["bookmarks"])):
+            if response["bookmarks"][item]["title"] == "Incident report":
+                document_id = extract_google_doc_id(response["bookmarks"][item]["link"])
+
+    # Update the document status to "Closed" if we can get the document
+    if document_id != "":
```

Comment:
Just a minor request - could you log a warning here if the Google doc can't be found.